tests/Feature/ReviewTableViewTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace LaravelViews\Test\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use LaravelViews\Test\Database\FoodTest;
+use LaravelViews\Test\Database\ReviewTest;
+use LaravelViews\Test\Database\UserTest;
+use LaravelViews\Test\Mock\MockReviewTableView;
+use LaravelViews\Test\Mock\MockReviewTableViewWithSearch;
+use LaravelViews\Test\Mock\MockTableViewWithSearchAndFilters;
+use LaravelViews\Test\TestCase;
+use Livewire\Livewire;
+
+class ReviewTableViewTest extends TestCase
+{
+    use RefreshDatabase;
+
+
+    private $livewire;
+    private $users;
+    private $reviews;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        factory(FoodTest::class, 10)->create();
+        $this->users = factory(UserTest::class, 10)
+            ->create()
+            ->each(function (UserTest $user) {
+                $user->reviews()->saveMany(factory(ReviewTest::class, 1)->make());
+            });
+
+        $this->reviews = ReviewTest::all();
+
+        $this->livewire = Livewire::test(MockReviewTableView::class);
+    }
+
+    public function testSeeAllHeaders()
```

Comment:
I'm not sure about these methods, they are replicated in the table view tests, in these test file we want to be focus on the relational search, the `testSeeAllDataFoundBySearchInput ` might be enough, in that case we could make it even stronger with more test cases, what do you think?

Reply:
I had my doubts as well, I will remove the duplicate tests.